README.md

```diff
@@ -218,27 +220,37 @@ or
     /var/log/uwsgi.log
 
 ## Create Bike facility overlays from OSM file
+    These steps are going to depend on your region
+    Set the OSM_REGION_FILE as needed:
+    ```
+    export OSM_REGION_FILE="bayarea.osm"
+    export OSM_REGION_FILE="tahoe.osm"
+    ```
+    Also, for tahoe, change `bicycle=designated,yes` to only `bicycle=designated`. The OSM data in Tahoe has been cleaned such that all ways that we want to include on the map are tagged with `bicycle=designated.`
+
+    ```
+    export ALLOWED_BIYCLE_TAGS="desginated,yes"
+    export ALLOWED_BIYCLE_TAGS="designated"
 
-    osmosis --read-xml bayarea.osm --tf accept-ways highway=* --tf reject-ways surface=dirt,grass,clay,sand,earth,pebblestone,ground,grass_paver,unpaved,woodchips,snow,ice,salt --write-xml bikefacilities.osm
```

Comment:
I deleted this command, because it seemed redundant. Higher up, when creating bayarea.osm we already apply these same filters:
```
~/downloads/osmosis/bin/osmosis --read-xml california-latest.osm --bounding-box left=-122.769606 bottom=37.459723 right=-121.611723 top=38.064476 completeWays=yes --tf accept-ways highway=* --tf reject-ways surface=dirt,grass,clay,sand,earth,pebblestone,ground,grass_paver,unpaved,woodchips,snow,ice,salt --write-xml bayarea.osm
```

This ended up just being a rename operation. Instead I handle this with a variable for the filename.

The upside here is we can apply different filters in each region.